src/App.vue

```diff
@@ -417,6 +418,12 @@ export default class App extends Vue {
     return !this.$route.meta.noStepper
   }
 
+  /** The current domain. */
+  get domain (): string {
+    // remove possible leading period
+    return window.location.hostname.replace(/^\./, '')
+  }
```

Comment:
This value is used 3 times in this component, and needed some extra logic to handle some domains, for example:
- `localhost`
- `.business-create-dev.apps.silver.devops.gov.bc.ca`